src/test/resources/catalogs/default/datatypes/T_INT.ion

```diff
@@ -1,26 +0,0 @@
-{
```

Comment:
self-review: PLK 0.14.9 supported Ion's arbitrary int. Since PLK-1.x no longer supports arbitrary ints, this table definition no longer needed. We already support the other Int types like int16 and int32